src/ebonite/build/docker.py

```diff
@@ -80,6 +79,15 @@ class DockerHost(RuntimeEnvironment.Params):
     def __init__(self, host: str = ''):
         self.host = host
 
+    def get_runner(self):
+        """
+        :return: docker runner
+        """
+        if self.default_runner is None:
+            from ebonite.build import DockerRunner
+            self.default_runner = DockerRunner()
```

Comment:
`DockerRunner` knows about `DockerHost`, `DockerHost` knows about `DockerRunner`. Looks like we love to repeatedly shoot ourselves in foot. OK, at least we are consistent with `Task.push_model`.

Reply:
that's a good point, but we stuck between two evils - type hints but with strange circular imports or local imports, and it seems there is no third option. but in this particular case i think there is - `DockerHost` is "runtime" entity and only used in `ebonite.build.runner.docker` module, so we can move it there - and get rid of that local import. And I also proposed to move it all to `ebonite.ext` in ENBT-268

Reply:
I meant we can leave it like this for now and change while working on EBNT-268

Reply:
meh, ok